<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainView</class>
 <widget class="QMainWindow" name="MainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainView</string>
  </property>
  <property name="windowIcon">
   <iconset resource="clientresources.qrc">
    <normaloff>:/root/images/Images/logo.png</normaloff>:/root/images/Images/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnConnect">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDisconnect">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbNoiseParameters">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>281</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Noise Parameters</string>
    </property>
    <widget class="QGroupBox" name="gbOpenNoise">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Open Noise</string>
     </property>
     <widget class="QTextEdit" name="editOpenNoise">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gbCloseNoise">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>101</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Close Noise</string>
     </property>
     <widget class="QTextEdit" name="editCloseNoise">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbData">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>761</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Data</string>
    </property>
    <widget class="QPushButton" name="btnSend">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="gbDataParameters">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Data Parameters</string>
     </property>
     <widget class="QTextEdit" name="editDataParameters">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gbMessage">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>491</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Message</string>
     </property>
     <widget class="QTextEdit" name="editMessage">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>451</width>
        <height>41</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTextEdit" name="editServerResponse">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>761</width>
      <height>111</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>808</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>editOpenNoise</tabstop>
  <tabstop>editCloseNoise</tabstop>
  <tabstop>editDataParameters</tabstop>
  <tabstop>editMessage</tabstop>
  <tabstop>btnSend</tabstop>
  <tabstop>btnDisconnect</tabstop>
  <tabstop>btnConnect</tabstop>
  <tabstop>editServerResponse</tabstop>
 </tabstops>
 <resources>
  <include location="clientresources.qrc"/>
 </resources>
 <connections/>
</ui>
